rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -684,6 +686,14 @@ public String corsDomains() {
         return configFromFiles.getString(PROPERTY_RPC_CORS);
     }
 
+    public Integer getMaxPayloadSize() {
```

Comment:
```suggestion
    public int getMaxPayloadSize() {
```
`configFromFiles.getInt()` returns `int`. Seems no need to convert it to an `object`